### PR TITLE
Loading Page UI

### PR DESCRIPTION
**Pull request by bycho91**
- Note: Not hooked to the BE just yet. This is only the UI
- The Loading page will only load on initial Sign In/Sign Up when user is being routed to the Home.tsx page. It will show for 5 seconds and then proceed to show the budget cards
- I commented out the Close modal button for now since I wasn't sure what the intention behind it was. Modal will auto close after 5 seconds as mentioned above

### CODE REVIEW

File: client/src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
     {}
   );
 
+  const [position, setPosition] = useState<number>(12);
+
+  useEffect(() => {
+    displayLoader &&
+      setTimeout(() => {
+        setDisplayLoader(false);
+      }, 3000);
```
Comment (Jmartimus) [inline client/src/pages/Home/Home.tsx:30]: You may need to add a cleanup function here.

**Review by Jmartimus — CHANGES_REQUESTED** [review]
Check out the 1 comment.

File: client/src/pages/Home/Home.tsx (hunk 1 above)
Comment (bycho91) [reply client/src/pages/Home/Home.tsx:30]: Never done this before haha but I changed it - let me know if it's okay now

**Review by Jmartimus — APPROVED** [review]
LGTM

**Pull request merged**